Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationInfoCard.kt

```diff
@@ -39,6 +39,6 @@ class VaccinationInfoCard(parent: ViewGroup) :
         val daysUntilImmunity: Int?
     ) : CertificateItem, HasPayloadDiffer {
         override fun diffPayload(old: Any, new: Any): Any? = if (old::class == new::class) new else null
-        override val stableId = daysUntilImmunity.hashCode().toLong()
+        override val stableId = Item::class.hashCode().toLong()
```

Comment:
it was originally this way ,but we used the daysUntilImmunity to update the card when time changes

Reply:
Hm, the hashcode of INT is the INT itself, isn't it?

I think it might have been the cause for this seemingly random crash:
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8230

if the time changes, the `stableId` should stay fixed though for the info card, as it is still the same item, just it's content changed.

Reply:
We do use the latest payload too:
```
        val curItem = payloads.filterIsInstance<Item>().singleOrNull() ?: item
        val daysUntilImmunity = curItem.daysUntilImmunity
```
are we sure that it doesn't work?

Reply:
Works

Reply:
I've tried it.
There is no timer on that screen that would cause a live update, but if there would be, the value would change if the  `data class Item` is updated.